sys/linux/socket_netlink_route.txt

```diff
@@ -850,6 +869,40 @@ br_mdb_entry_sub_u [
 	in6_addr	ipv6_addr
 ]
 
+rtm_nh_policy_get [
+	NHA_ID		nlattr[NHA_ID, int32[0:2]]
+	NHA_OIF		nlattr[NHA_OIF, ifindex]
+	NHA_GROUPS	nlattr[NHA_GROUPS, void]
+	NHA_MASTER	nlattr[NHA_MASTER, int32[0:2]]
+	NHA_FDB		nlattr[NHA_FDB, void]
+] [varlen]
+
+rtm_nh_policy_new [
+	NHA_ID		nlattr[NHA_ID, int32[0:2]]
+	NHA_GROUP	nlattr[NHA_GROUP, array[nexthop_grp]]
+	NHA_GROUP_TYPE	nlattr[NHA_GROUP_TYPE, flags[nha_group_types, int16]]
+	NHA_BLACKHOLE	nlattr[NHA_BLACKHOLE, void]
+	NHA_OIF		nlattr[NHA_OIF, ifindex]
+	NHA_GATEWAY	nlattr[NHA_GATEWAY, nha_gateway]
+	NHA_ENCAP_TYPE	nlattr[NHA_ENCAP_TYPE, flags[lwtunnel_encap_types, int16]]
+	NHA_ENCAP	nlnest[NHA_ENCAP, lwtunnel_policy]
+	NHA_FDB		nlattr[NHA_FDB, void]
+] [varlen]
+
+type rtm_nh_policy_del nlattr[NHA_ID, int32[0:2]]
+
+nexthop_grp {
+	id	int32[0:2]
```

Comment:
We have `int32[0:2]` here, and in three places (`NHA_ID`) above. 

They correspond to the same thing, right? (maybe `nexthop_grp.id` is different but others (`NHA_ID`) seemed to mean the same).

Just like [here](https://github.com/google/syzkaller/blob/52017a407ace34e160c0bae1cb465b606b1fe51d/sys/linux/socket_netlink_route.txt#L36), we can define rtm_nh_id once and use it here and there.

Besides that, LGTM!

Reply:
Ah yes, I had that before, reverted to include it again, thanks!